vispy/scene/cameras/base_camera.py

```diff
@@ -79,28 +79,35 @@ def __init__(self, interactive=True, flip=None, up='+z', parent=None,
         # available in all cameras. Note that PanZoom does not use _center
         self._fov = 0.0
         self._center = None
+        self._depth_value = 1e6  # bit+depth >= 24, otherwise should do 3e3
 
         # Set parameters. These are all not part of the "camera state"
         self.interactive = bool(interactive)
         self.flip = flip if (flip is not None) else (False, False, False)
         self.up = up
 
-    def _get_depth_value(self):
-        """ Get the depth value to use in orthographic and perspective projection
+    @property
+    def depth_value(self):
+        """The depth value to use  in orthographic and perspective projection
 
```

Comment:
What IS the depth value? What are its practical effects? I assume it somehow relates to the distance to the far clipping plane?